doc/design/OrionCKANSink.md

```diff
@@ -1,50 +1,63 @@
+###<a name="section1.2"></a>Mapping Flume events to CKAN data structures
+[CKAN organizes](http://docs.ckan.org/en/latest/user-guide.html) the data in organizations containing packages or datasets; each one of these packages/datasets contains several resources whose data is finally stored in a PostgreSQL database (CKAN Datastore) or plain files (CKAN Filestore). Such organization is exploited by `OrionCKANSink` each time a Flume event is going to be persisted.
 
-In addition, when running in `column` mode, due to the number of notified attributes (and therefore the number of fields to be written within the Datastore) is unknown by Cygnus, the Datastore can not be automatically created, and must be provisioned previously to the Cygnus execution. That's not the case of the `row` mode since the number of fields to be written is always constant, independently of the number of notified attributes.
+According to the [naming conventions](naming_convetions.md), an organization called as the `fiware-service` header values is created (if not existing yet). The same occurs with a package/dataset named as the `fiware-servicePath` header value.
+
+The context responses/entities within the container are iterated, and a resource named as the `destination` header value is created (if not yet existing). A datastore associated to the resource is created as well.
+
+The context attributes within each context response/entity are iterated, and a new data row is upserted in the datastore related to the resource. The format for this append depends on the configured persistence mode:
+
+* `row`: A data row is upserted for each notified context attribute. This kind of row will always contain 8 fields:
+    * `recvTimeTs`: UTC timestamp expressed in miliseconds.
+    * `recvTime`: UTC timestamp in human-redable format ([ISO 8601](http://en.wikipedia.org/wiki/ISO_8601)).
+    * `entityId`: Notified entity identifier.
+    * `entityType`: Notified entity type.
+    * `attrName`: Notified attribute name.
+    * `attrType`: Notified attribute type.
+    * `attrValue`: In its simplest form, this value is just a string, but since Orion 0.11.0 it can be Json object or Json array.
+    * `attrMd`: It contains a string serialization of the metadata array for the attribute in Json (if the attribute hasn't metadata, an empty array `[]` is inserted).
+* `column`: A single row is upserted for all the notified context attributes. This kind of row will contain two fields per each entity's attribute (one for the value, called `<attrName>`, and other for the metadata, called `<attrName>_md`), plus four additional fields:
+    * `recvTime`: UTC timestamp in human-redable format ([ISO 8601](http://en.wikipedia.org/wiki/ISO_8601)).
+    * `fiwareservicePath`: The notified one or the default one.
+    * `entityId`: Notified entity identifier.
+    * `entityType`: Notified entity type.
```

Comment:
Any refactor in this part of the doc? Or there are relevant additions/deletions to delete? The diff is a bit noisy... :)